pages/pipelines/environment_variables.md.erb

```diff
@@ -303,6 +303,18 @@ The organization name on Buildkite as used in URLs. The value cannot be modified
 
 Example: `"acme-inc"`
 
+<h3 class="h3-caps">BUILDKITE_PARALLEL_JOB</h3>
+
+The number of parallel jobs created from a parallel build step. Starting from 0. For a build step with `parallelism: 5`, the value would be 0, 1, 2, 3, and 4 respectively.
+
+Example: `"0"`
+
+<h3 class="h3-caps">BUILDKITE_PARALLEL_JOB_COUNT</h3>
+
+The total number of parallel jobs created from a parallel build step. For a build step with `parallelism: 5`, the value is 5.
```

Comment:
This is the max parallel jobs right? There could be a case where there are only 2 parallel jobs running but there theoretically could be 5? Or is it the actual count of parallel jobs created?

Reply:
If `parallelism` says 5 it should be 5 for that step. Otherwise, something went wrong 😅